### DISCUSSION TITLE
search.google.com does not indexing my wiki

### DISCUSSION THREAD

**Discussion by steam3d** [docsifyjs/docsify#1581] [General]
How to add my wiki to search engines?

**Comment by trusktr** [comment]
This is currently a limitation if Docsify: it runs dynamically on the client side (in your browser) and because of this there is no HTML response from a server for Google to index, and Google does not know (does not intend to support) indexing of dynamic hash-based routed applications (where the URL path starts with `#`).

We have a solution for this in the works. There will be optios for serving a Docsify site with SSR (server side rendering), or precompiling it with SSG (static site generation).

With SSR, you would host a server that serves all your Docsify pages as HTML.

With SSG, you don't need a server, and there will be an initial static HTML page for every route.

In both cases, once the website is loaded in the browser, the dynamic nature of the markdown loading can take over.

Stay tuned...
